packages/test-cases/cases/bind/sanity/output/wasm-as/mutation/Mutation/serialization.ts

```diff
@@ -36,14 +38,22 @@ export function deserializemutationMethodArgs(argsBuf: ArrayBuffer): Input_mutat
     numFields--;
     const field = reader.readString();
 
+    reader.context().push(field, "unknown", "searching for property type");
     if (field == "str") {
+      reader.context().push(field, "string", "type found, reading property");
       _str = reader.readString();
       _strSet = true;
+      reader.context().pop();
+      reader.context().pop();
```

Comment:
This second "pop" for the "searching for property type" message above can be moved below the if / else if chain. This will make sure that the "searching..." message doesn't get left on the stack if the name of the property is unknown.

Making a push now with this update.